### PR TITLE
Make FunctionList sorting setting memorized between sessions

### PR DESCRIPTION
**Pull request by Kataderon**
Here added new setting item which switches up permanent sorting of function list.

### CODE REVIEW

**Comment by donho** [comment]
@Kataderon What do you mean by `switches up permanent sorting of function list` ?

**Comment by chcg** [comment]
@Kataderon @donho Guess switch on/off sorting is meant.

**Comment by MAPJe71** [comment]
@Kataderon is this the/your solution/implementation for issue(s) #98 and/or #1151?

File: PowerEditor/src/Parameters.cpp [hunk 1]
```diff
@@ -5069,8 +5069,8 @@ void NppParameters::feedGUIParameters(TiXmlNode *node)
 			if (optNameBackSlashEscape && !lstrcmp(optNameBackSlashEscape, TEXT("no")))
 				_nppGUI._backSlashIsEscapeCharacterForSql = false;
 
-			const TCHAR * optFLSort = element->Attribute(TEXT("flSort"));
-			if (optFLSort && !lstrcmp(optFLSort, TEXT("no")))
+			const TCHAR * optFunctionListAlwaysSort = element->Attribute(TEXT("functionListAlwaysSort"));
+			if (optFunctionListAlwaysSort && !lstrcmp(optFunctionListAlwaysSort, TEXT("no")))
 				_nppGUI._functionListSort = false;
```
[the remark this answers is not in the document]
Comment (Kataderon) [reply PowerEditor/src/Parameters.cpp]: I tried to fix it.

**Comment by Kataderon** [comment]
> @Kataderon is this the/your solution/implementation for issue(s) #98 and/or #1151?

Yes, it's my solution.

**Comment by donho** [comment]
@Kataderon 
Could youd solve the conflict please?

**Comment by Kataderon** [comment]
I solved conflict. But I have problem with squashing commits. Is there another way to make my pr suit?

**Comment by donho** [comment]
make a new branch from the current master locally and do a new PR.

**Pull request closed without merging**